lnwallet/wallet.go

```diff
@@ -484,15 +484,16 @@ func (l *LightningWallet) handleFundingReserveRequest(req *InitFundingReserveMsg
 	if req.LocalFundingAmt != 0 {
 		// Coin selection is done on the basis of sat/kw, so we'll use
 		// the fee rate passed in to perform coin selection.
-		err := l.selectCoinsAndChange(
+		coins, change, _, err := l.selectCoinsAndChange(
 			req.FundingFeePerKw, req.LocalFundingAmt, req.MinConfs,
-			reservation.ourContribution,
 		)
 		if err != nil {
 			req.err <- err
 			req.resp <- nil
 			return
 		}
+		reservation.ourContribution.Inputs = coins
+		reservation.ourContribution.ChangeOutputs = change
```

Comment:
Unrelated: any reason this is a slice rather than just one?

Reply:
Not sure. Legacy reasons I guess